front/SettingsForm.tsx

```diff
@@ -81,6 +82,7 @@ const SettingsForm = () => {
               className="form-control form-control-lg"
               rows={8}
               placeholder="Short bio about you"
+              data-cy = 'settings-bio'
```

Comment:
Remove the space between 'data-cy' and '='. It should be written as 'data-cy="settings-bio"'